packages/time-picker/src/vaadin-time-picker.js

```diff
@@ -394,6 +395,16 @@ class TimePicker extends PatternMixin(InputControlMixin(ThemableMixin(ElementMix
     }
   }
 
+  /**
+   * Override an event listener from `KeyboardMixin`
```

Comment:
**nit:** Could be helpful to know from where exactly the listener we are overriding.

```suggestion
   * Override an event listener from `InputControlMixin`
```